source_integration_test.go

```diff
@@ -28,10 +27,13 @@ import (
 
 func testSource(ctx context.Context, is *is.I, cfg config.Config) (sdk.Source, func()) {
 	source := &Source{}
-	cfg[common.SourceConfigDsn] = testutils.DSN
-	cfg[common.SourceConfigDisableCanalLogs] = "true"
 
-	err := source.Configure(ctx, cfg)
+	cfg["dsn"] = testutils.DSN
+	cfg["disableCanalLogs"] = "true"
+
+	err := sdk.Util.ParseConfig(
+		ctx, cfg, source.Config(), Connector.NewSpecification().SourceParams,
+	)
```

Comment:
I think it should be possible to do `source.Config().(common.SourceConfig)` and then set the parameters directly. That might simplify the test code, since it would get rid of the maps.

Reply:
Yeah, you right, thanks

Reply:
Fixed